main.c

```diff
@@ -229,6 +233,33 @@ static bool check_timeouts(struct timeval *t)
 	return tmin > 0;
 }
 
+size_t win_title_get(char *buf, int len)
```

Comment:
We seem to be using `do_something` with other scripts. Similar to the operator thing, the codebase is not so consistent with naming, so we may leave this as-is (which is more consistent with `win_*` stuff).
```suggestion
size_t get_win_title(char *buf, int len)
```

Reply:
> which is more consistent with `win_*` stuff

Yeah, that was my intention. I'm okay changing it to `get_win_title` if we prefer that.